src/index.d.ts

```diff
@@ -42,10 +42,16 @@ export function useSSR(initialI18nStore: any, initialLanguage: any): void;
 export interface UseTranslationOptions {
   i18n?: i18next.i18n;
 }
+export interface UseTranslationResponse extends Array<i18next.TFunction | i18next.i18n> {
```

Comment:
@rosskevin @schettino I've no clue about typescript... Is this ok?

Reply:
@adrai you can try:

```ts
type UseTranslationResponse = [i18next.TFunction, i18next.i18n] & {
  t: i18next.TFunction;
  i18n: i18next.i18n;
}
```

Reply:
thx, just updated it

Reply:
This is bizarre - but I confirmed it typechecks correctly.